flowmachine/flowmachine/core/server/utils.py

```diff
@@ -53,11 +53,14 @@ def send_zmq_message_and_receive_reply(msg, port=5555, host="localhost"):
 FM_EXAMPLE_MESSAGE = {
     "action": "run_query",
     "params": {
-        "query_kind": "daily_location",
-        "date": "2016-01-01",
-        "method": "last",
-        "aggregation_unit": "admin3",
-        "subscriber_subset": None,
+        "query_kind": "spatial_aggregate",
+        "locations": {
```

Comment:
It just occurred to me that when reading this spec, I expected the value of the `"locations"` key to be a list because "locations" is plural. I don't think making it singular makes sense, but something to keep in mind because it may lead to confusion. Maybe we want to rename it to `"locations_query"` in the future?

Reply:
Maybe, yeah. I had exactly that originally, but decided that because it is the locations of multiple _subscribers_ that the plural made sense.

Reply:
Yes, I feel the plural in locations is appropriate. Would just be nice from a user's perspective to have a name that makes it reasonably obvious what kind of value is expected.